README.md

```diff
@@ -362,6 +362,9 @@ py_image(
 )
 ```
 
+You can also put all external dependencies to a separate layer at the bottom,
+by setting `external_layer=True`.
```

Comment:
This isn't specific to `py_image`, and isn't really "fine layering".  Perhaps add a new example illustrating how to do this grouping all external deps?